low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -23,39 +23,41 @@ import sys
 import socket
 import json
 import subprocess
+import tempfile
+import warnings
+warnings.filterwarnings(action='ignore',module='cryptography')
```

Comment:
Please move this below all imports. Also add a space after comma.

Reply:
Done